lib/src/cluster_manager.dart

```diff
@@ -80,8 +80,7 @@ class ClusterManager<T extends ClusterItem> {
   void _updateClusters() async {
     List<Cluster<T>> mapMarkers = await getMarkers();
 
-    final Set<Marker> markers =
-        Set.from(await Future.wait(mapMarkers.map((m) => markerBuilder(m))));
+    final Set<Marker> markers = Set.from(await Future.wait(mapMarkers.map((m) => markerBuilder(m))));
```

Comment:
Please make some adjustments to code formatting to reduce line length. It's out of scope